.gitpod.dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM gitpod/workspace-mysql
 
-# Install Ruby version 2.4.6 and set it as default
+# Install Ruby version 2.7.6 and set it as default
 RUN echo "rvm_gems_path=/home/gitpod/.rvm" > ~/.rvmrc
-RUN bash -lc "rvm install ruby-2.4.6 && rvm use ruby-ruby-2.4.6 --default"
+RUN bash -lc "rvm install ruby-2.7.6 && rvm use ruby-ruby-2.7.6 --default"
```

Comment:
Here, getting:

```

Step 3/4 : RUN bash -lc "rvm install ruby-2.7.6 && rvm use ruby-ruby-2.7.6 --default"
 ---> Running in b049f1523d7b
Already installed ruby-2.7.6.
To reinstall use:

    rvm reinstall ruby-2.7.6

Gemset '' does not exist, 'rvm ruby-2.7.6 do rvm gemset create ' first, or append '--create'.
The command '/bin/sh -c bash -lc "rvm install ruby-2.7.6 && rvm use ruby-ruby-2.7.6 --default"' returned a non-zero code: 2
```

Dunno why... why is 2.7.6 already installed?

Reply:
I followed steps here to find the error: https://github.com/gitpod-io/gitpod/issues/9875

doing `rvm uninstall ruby-2.7.6` didn't work to get it to pass... :-(